module_intros/diffraction.StaticStructureFactor.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<ol><li>I would use lowercase letters r_i, r_j.</li><li>Fix typo: "integrating"</li><li>I would avoid saying that the Debye method integrates over <strong><em>orientations</em></strong>, because that suggests to me that it has to do with particle orientations (quaternions). I suggest "integrating over a spherical shell" or simply "integrating out angular dependence"? Quaternions exist on the 3-sphere (like Euler angles alpha, beta, gamma) because of their additional degree of freedom, while we are only integrating over a 2-sphere (theta, phi).</li><li>I would clarify the conventions by specifying that the alternative is the crystallographic convention. The formula $k = 2pi/r$ isn't quite accurate, so I would remove that.</li></ol><blockquote>Note that freud employs the physics convention for reciprocal vectors, $\vec{k} \cdot \vec{r}$ is a multiple of $2\pi$ for a lattice vector $\vec{r}$ and corresponding reciprocal lattice vector $\vec{k}$. The crystallographic convention typically uses the letter $\vec{q}$, and is related by $\vec{k} = 2 \pi \vec{q}$.</blockquote><p>Alternatively we might use the exponential notation that is common in physics, like $e^{i \vec{k} \cdot \vec{r}} = 1$ and $e^{2 \pi i \vec{q} \cdot \vec{r}} = 1$.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>The Fourier transform lacks the imaginary unit in the exponential.</p>